docsrc/index.rst

```diff
@@ -4,44 +4,29 @@
 What is Cyrus IMAP?
 ===================
 
-Cyrus IMAP is an email, contacts and calendar server. Cyrus is free and open source.
+Cyrus IMAP is an efficient, reliable, and scalable server for email, calendar,
+and and contacts server. Cyrus is free and open source.
 
+Cyrus supports JMAP, IMAP, POP3, CalDAV, CardDAV, NNTP, and
+:doc:`many other standards <imap/rfc-support>`.  It supports Sieve for
+filtering email. You can read more about Cyrus features in our full :ref:`list
+of features <imap-features>`.
 
 * This is version |imap_latest_development_version|: **development** branch.
 * The latest stable version |imap_current_stable_version| is at :cyrus-stable:`/`.
 
 .. Update this when there is a changeover from stable to development.
 
-Features
---------
-
-* Speed
-* Security
-* Efficiency
-* Search
-* CardDAV and CalDAV support
-* Robust data storage
-* Replication
-* Scalable for large installations
-* Flexible filtering support through Sieve
-* Supports antivirus and antispam toolkits
-* Handles deliverability: SPF, DKIM, DMARC
-* Extensive authentication options, through Cyrus SASL
-* JMAP support
-* Long list of supported standards
-
-Read more in our full :ref:`list of features <imap-features>`.
-
-Cyrus has been under active development since the year 1993 when the project was launched at Carnegie Mellon University. It's used in production
-systems around the world, at universities and in private enterprise.
+Cyrus IMAP supports many authentication options.  It uses Cyrus SASL, an
+implementation of the `Simple Authentication and Security Layer (SASL)
+<https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer>`_
+system.  You can read more at the `Cyrus SASL web page <http://www.cyrusimap.org/sasl>`_.
```

Comment:
You might be able to link to this using an RST thingy rather than a direct hyperlink, since the two repositories get built into a single website, and both of them publish a TOC thingy so they should know what each other's main headings are.  Especially since this file is "docsrc/index.rst" (and not "docsrc/imap/index.rst"), and is therefore supposed to be the front page for the entire common website.  There might be a magic word defined in docsrc/conf.py that you can invoke.

In practice, I would look for other references to the SASL site and see how they're linked, and do whatever they do.   Maybe you already did that and this is how it's done.

Reply:
Hrm.  I will do this, but I don't think it's actually a good idea for us to do this.  I will try to suck it up, it's probably fine.  This works:

```
:ref:`Cyrus SASL <cyrussasl:sasl-index>`
```

…not because they are built into a single website, but because `cyrussasl` is configured as an "intersphinx link", which means that when we build the Cyrus IMAP website, it will fetch, *via HTTP*, the TOC of the published SASL site.  So, there's sort of an ordering / bootstrapping issue.

Whatever.  We can fix it someday or not.  For now, we use it, so I've done it to be consistent.